editoast/src/views/pagination.rs

```diff
@@ -65,16 +66,49 @@ fn default_page() -> i64 {
     1
 }
 
+impl PaginationQueryParam {
```

Comment:
There's a bunch of `25` hanging around, better define `const DEFAULT_PAGE_SIZE: i64 = 25;`